### ISSUE TITLE
[CAIA intake] Updates to 10-10CG Caregiver Application

### ISSUE THREAD

**Issue by jessicastump** [department-of-veterans-affairs/va.gov-team#81190]
# Content, accessibility, information architecture (CAIA) new initiative collaboration request

Use this ticket to start collaboration on a new initiative with the sitewide content, accessibility, and information architecture (CAIA) team.

**Note:** If you’re already partnering with us on an initiative, you don't need to fill out this request form. If you don’t have access to your initiative’s CAIA epic, post a message in the [#sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-acessibility-ia) slack channel and tag `@Terry Nichols` and `@Lily Strelich`.

- Met with @aliyahblackmore during weekly 1010 team content sync to review, advised to open request intake ticket
- cc @hdjustice 

## About your team

- **Team name**: 10-10 Health Enrollment Team
- **OCTO product owner:** Patrick Bateman
- **Product name:** 10-10CG Caregiver Application
- **Product manager:**  Heather Justice
- **Slack channel:** #1010-health-apps 
- **Dedicated content writer on your team (if you have one):** Jessica Stump ([@jessicastump](https://github.com/jessicastump)) and Angela Agosto ([@aagosto90](https://github.com/aagosto90))
- **Dedicated a11y specialist on your team (if you have one):** N/A
 
### Which of these descriptions best fits the work we’ll partner on?
Select all that apply.
- [ ] Digitizing a new form
- [ ] Creating a new digital tool
- [x] Updating an existing form or tool
- [ ] Translating a form or tool into Spanish
- [ ] Adding new unauthenticated content to VA.gov
- [ ] Updating existing unauthenticated content on VA.gov
- [ ] Something else:  `insert description`

### What's the nature of your initiative and desired outcomes?

We need to make some updates to the online form to align with the new paper form and program updates.

## Collaboration timeframe

**Note:** We work on nearly every OCTO product and manage all unauthenticated content on VA.gov, so we will need to prioritize intake requests based on overall workload and VA and OCTO priorities.

### Is this work tied to a Congressional mandate, change in law or policy, or upcoming event with a specific deadline?
- [ ] Yes
- [ ] No
- [x] I’m not sure

### Where are you at in your timeline?
Tell us briefly about what you're working on now (such as initial discovery, wireframing, or usability research planning) and add any known dates for upcoming milestones or deadlines.

- We have received the new paper form updates and are working with stakeholders making updates to the back end systems and schema. We have updated user flows in Mural and annotated changes ready to go in Figma. We have walkthrough the requested changes with our content person Aliyah during our weekly content/1010 team sync.

- Need to have content finalized by **EARLY MAY 2024** to meet development/QA timeline

### Which phases of the collaboration cycle have you completed?
Select all that apply.
- [ ] Design intent 
- [ ] Midpoint review 
- [ ] Staging 
- [x] None. This initiative isn’t going through the collaboration cycle.

### Collaboration cycle ticket 

If you’re going through the collaboration cycle, provide your ticket number and link:
- **Ticket Number:** `xxxxxx`

## Supporting artifacts
Provide links to any supporting artifacts that can help us better understand your initiative and begin collaboration. Include artifacts like your product outline, user flows, mockups and prototypes, or any draft content.
- [User flow in Mural](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1711491442696/6c33e417dbfd2cb893452606262192fcb79a579b?sender=uadf1ed7fe7c76f0914967329) 
- [Figma designs for content review](https://www.figma.com/file/TxXD5bGUOhbHHWLb85GPjK/10-10CG?type=design&node-id=376-15166&mode=design&t=PfcGzs31wPaIXZBG-0) 
- [GitHub Project Folder](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/caregivers/Form%20updates%20(Feb%202024))

## Next steps
- [ ] Post a link to this issue in the [#sitewide-content-sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-accessibility-ia) slack channel and tag `@Terry Nichols`. We’ll reply within 24 hours to acknowledge receipt. We'll then review the artifacts in more detail and work with you to determine timing for collaboration. Depending on the work, we may schedule a kickoff meeting to better understand how we'll partner together.
- [ ] Review the remaining next steps in the **VFS team tasks** section here.

## Tasks

<br>

```[tasklist]
### VFS team tasks
- [ ] **If you also need engineering support from the public websites team**, fill out their intake request form as well.<br> [Open a public websites intake request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E)
- [ ] **If you're requesting a page/URL redirect, a URL change, or a vanity URL**, submit an additional request for this work. <br> [Open a redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=jennymayoco%2C+kristinoletmuskat%2C+strelichl%2C+FranECross&labels=sitewide+CAIA%2C+Sitewide+IA%2C+Public+Websites%2C+VA.gov+frontend%2C+Redirect+request&projects=&template=redirect-request.md&title=Redirect+Request)
- [ ] **If you're creating an experimental design**, also contact the design system team. <br> [Read more about experimental designs](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) <br> [Suggest an addition or update to the design system team](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)
- [ ] **If you're requesting a11y support for usability research**, submit a research support request next. <br> [Open an a11y research support request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name)
- [ ] **If you are requesting content support**, please share your content source of truth (preferably as a SharePoint document).
```
```[tasklist]
### Content tasks
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/83893
```

**Comment by aliyahblackmore** [comment]
**Cross posting Slack updates here** 

- I'll continue to go through the CG screens today and make adjustments and add questions. But since there are some outstanding SME questions that may impact content, I'll plan to return to the CG designs when I'm back in next week and make updates.
- Jessica, I know you mentioned an early May deadline for having design/content ready - so I'll plan (TBD based on SME answers) to have content any outstanding content updates ready by Wednesday May 8.

**Comment by aliyahblackmore** [comment]
**Update here:** 

This is still in progress. 

- I chatted with product team about Figma edit and we decided to edit in a different section. I'll need to track my changes in that section. 

- And the product team has a few SME and design system questions that are still outstanding.

- I'll continue on Monday.

**Comment by jessicastump** [comment]
@aliyahblackmore - updating here too: the 3 items that are outstanding design system questions were posed in the Slack channel and after talking at the product ux sync, I RSVP'd to Platform Design Office hours next week on the 21st. I looked over the Figma file and comments again and there should be no more outstanding SME questions. For the component/section issue in Figma - the change seems to have fixed that, but let me know if you still see any issues!

**Comment by aliyahblackmore** [comment]
Thanks for the update @jessicastump ! 

I'll continue tracking content in the source of truth areas today and I'll flag if there are still component/section issues in Figma.

**Comment by aliyahblackmore** [comment]
**Update here:**

- During design/content sync we confirmed a few other details about the form.
- I'll wrap up the last small content adjustments on screens and adjust the confirmation screen on Monday. 
- Team is planning to share designs with SMEs by end of next week.

**Comment by aliyahblackmore** [comment]
**Update here (cross posting from Slack):** 

- There are new replies to comments earlier in the form from a few minutes ago - I can take a look tomorrow.
- There are comments re: the statements of truth. My comment was based on the content teams conversation/decision re: PDFs and online form statements of truth being a 1:1 match. I'll ask the content team about this tomorrow and confirm an answer.
- I updated the content and SME questions/notes re: the VA forms 21-0972, 21-22, and 21-22a that are on the new PDF. Those VA forms don't prove that someone has legal authority to make decisions of behalf of a Veteran - so I need SMEs to confirm a question on this so that I can adjust the content accordingly.

**Comment by aliyahblackmore** [comment]
**Updates here (cc @strelichl ):**
- Content feedback and questions for SMEs are in Figma. The product team is sharing with their SMEs for their review.
- Additional content adjustments TBD based on SME feedback.
- Next I'll turn to the static page updates.

**Comment by aliyahblackmore** [comment]
Just updating here  @strelichl that the product team shared today that they'll likely launch the updated form July 1.

**Comment by aliyahblackmore** [comment]
**Cross posting an update here to close the loop (cc @jessicastump ):**

As long as your PO is okay with the stakeholder recommendation to remove the "If the Veteran is already enrolled in this program and wants to add a new family caregiver, the Veteran and the new caregiver can use this application" sentence from inside of the form (but leaving it on the form intro page and static page, that's okay.

And re: conversation around leaving the example from the updated PDF version of the form ("This includes a caregiver who's already designate...") off of the online form for the July 1 change, that's also okay as long as your SMEs/stakeholders and PO are aware of that difference between the online and PDF form.

And we can plan to revisit in the future if there are any regulation changes!

**Comment by aliyahblackmore** [comment]
Cross posting here from Slack/Figma threads re: the VA forms (cc: @jessicastump): 

- For the short term, it's okay to align more closely with the PDF and share that someone can submit one of these forms with their application. 
 - And we can plan to revisit the this at some point in the future! 

And I'll share the static page updates via email with the CG SMEs by EOD tomorrow.

**Comment by aliyahblackmore** [comment]
**Update here:**

I shared the static page updates with SMEs a few minutes ago and asked for an accuracy review.

**Comment by aliyahblackmore** [comment]
**Update here:** 

- On 6/17 and 6/18 CSP SMEs and stakeholders approved various sections of the PCAFC static page. 
- After further conversation on the 21-22/a "Note" we'll move forward with the SME/stakeholder request to remove the note entirely and to remove links to the VA form pages and include them in the same list as the other documents. 
- The CAIA concerns and content guidance recs are documented re: the "Note" and corresponding content. And we can revisit if the product team makes any future adjustments.

cc: @jessicastump

**Comment by aliyahblackmore** [comment]
**Update here:** 

- I sent an email to SMEs and stakeholders just confirming, as they requested, that we won't include a note about VA Form 21-22 or VA Form 21-22a. And the content aligns with the content they reviewed and approved in the online form (Jessica shared content update in their Friday 6/21 meeting). 
- And I asked them to just confirm that we are okay with this adjustment. And flagged that it's the same as what's on the online form.
- I outlined the static page updates they reviewed and approved previously. And confirm that the page will go through a copy edit, we'll publish it next week, and we'll share a link to the live page.

**Comment by aliyahblackmore** [comment]
**Cross posting updates here**

- SMEs gave us the okay to publish the VA.gov PCAFC page on 7/2. I'll send a message on Slack connecting the product team with OCTO folks who will just support publishing the page on 7/2.

- Jessica confirmed that the current link to the static page that's on the form intro page will redirect to the recently migrated/new PCAFC page in the family member benefits hub. The product team has a post 7/2 ticket to change out the URL, but this doesn't impact anything at this time. [This is the jump link](https://www.va.gov/family-and-caregiver-benefits/health-and-disability/comprehensive-assistance-for-family-caregivers/#what-documents-can-i-submit-if) to the section of the page that'll list the representative supporting documents. 


**And tracking some notes re: the VA forms conversation for awareness (nothing to do here):** 

* CG form added new PDF form options to the "Representative" section of online form. Content was formatted to include these new PDF forms and info on how to submit them.
* Aliyah spoke with A (managing editor) about these forms - they shared the research and concerns from the accredited rep teams — that research has resulted in their strong rec to include a note that warns users about what happens when they use accredited rep forms (this changes the rep for any other applications/claims/decision reviews).
* After sharing the initial note with SMEs and they raised concerns, Danielle suggested a meeting with A, Aliyah, and the 1010 team to determine a best next step.  Danielle recommended an adjusted note that reaches a sort of middle ground of the CG SME concerns outlined in their email thread and the concern and recommendation from the accredited rep teams. Aliyah shared the middle ground note and SMEs confirmed they don’t want to include it.
* SMEs shared new information on 6/21 - they can’t accept new VA forms only previously submitted/completed forms. At this time, they don’t want to include any other context or a note about these VA forms. 
* Aliyah confirmed this approach with Danielle - and this is fine. We can revisit any content on this in the future, if needed.

**Comment by aliyahblackmore** [comment]
These initial updates to the CG launched! Moving this to "Validate".

The product team can open a new post-research intake for any future updates.
